build.psm1

```diff
@@ -414,10 +414,10 @@ Fix steps:
 
     # handle ResGen
     # Heuristic to run ResGen on the fresh machine
-    if ($ResGen -or -not (Test-Path "$PSScriptRoot/src/Microsoft.PowerShell.ConsoleHost/gen")) {
-        Write-Log "Run ResGen (generating C# bindings for resx files)"
-        Start-ResGen
-    }
+    #if ($ResGen -or -not (Test-Path "$PSScriptRoot/src/Microsoft.PowerShell.ConsoleHost/gen")) {
```

Comment:
We should do more clean-up or file an issue to do it

Reply:
Yes, after MSFT team confirm that the PR is good I will clean up all scripts.